Client/WebAuthN/WebAuthnClientData.swift

```diff
@@ -0,0 +1,48 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import Shared
+
+enum  WebAuthnClientDataType: String {
+    case create = "webauthn.create"
+    case get = "webauthn.get"
+}
+
+struct WebAuthnClientData {
+    var type: String
+    var challenge: String
+    var origin: String
+    
+    enum CodingKeys: String, CodingKey {
```

Comment:
Redundant: CodingKeys here is synthesized by the compiler with the property names and since you didn't change their keys it will be the same

Reply:
I'm changing the types, `String, String, String` to `String, Data, String`. So I think we need `CodingKeys` here. Let me know if i'm missing something

Reply:
`CodingKeys` is unrelated to types and should exist even when you don't specify it yourself (its made implicitly by the compiler) for things that conform to `Codable`.  But it seems like this struct only conforms to `Encodable`, so you will actually need this yeah